dockers/docker-fpm-frr/Dockerfile.j2

```diff
@@ -62,6 +62,7 @@ RUN chmod a+x /usr/bin/TSA && \
     chmod a+x /usr/bin/zsocket.sh
 
 RUN j2 -f json /etc/rsyslog.d/rsyslog_plugin.conf.j2 /etc/rsyslog.d/events_info.json  > /etc/rsyslog.d/bgp_events.conf
+RUN sed -i '10i\$ModLoad omprog\' /etc/rsyslog.d/bgp_events.conf
```

Comment:
>$ModLoad omprog [](http://example.com/codeflow?start=16&length=15)

I am confused why you delete this line in rsyslog_plugin.conf.j2, but add back by `sed`.

If there is a good reason, you can still achieve it by j2 syntax, no need to have an extra sed command line.

Reply:
![image](https://github.com/sonic-net/sonic-buildimage/assets/99770260/fc24c90a-5891-420a-8585-132326195afe)

Reply:
Picture describes that modules can only be loaded once inside each tree. Red boxes show conf files generated by rsyslog plugin template.